examples/python/ghz.py

```diff
@@ -22,10 +22,9 @@
 import sys
 import os
 
-# We don't know from where the user is running the example,
-# so we need a relative position from this file path.
-# TODO: Relative imports for intra-package imports are highly discouraged.
-# http://stackoverflow.com/a/7506006
+# We don't know from where the user is running the example, so we need a
+# relative position from this file path. TODO: Relative imports for intra-
+# package imports are highly discouraged. http://stackoverflow.com/a/7506006
```

Comment:
In general, it would be safer to preserve the original comments when it comes to different "paragraphs" (in this case, some tools might require the `TODO` to be at the beginning of the line, and the URL in a separate line for quick reference).

Reply:
Hm, I have no Idea how this got on here. Most likely when I turned on word wrapping on my text editor. Will revert.